server/testUtils/App.ts

```diff
@@ -0,0 +1,45 @@
+import 'module-alias/register';
+import express from 'express';
+import getPort from 'get-port';
+import request from 'supertest';
+import { responseErrorHandler } from 'express-response-errors';
+import { Server } from 'http';
+
+type InitProps = {
+  withRouter?: express.Router;
+};
+
+class App {
+  server: express.Application;
+  request: request.SuperTest<request.Test>;
+  _server: Server;
```

Comment:
If this is supposed to be private (as is commonly denoted by the underscore), we could add private here so that TS enforces it for us ->  I.E. private _server: Server;